test/e2e/performanceprofile/functests/utils/cgroup/controller/controller.go

```diff
@@ -14,7 +14,10 @@ type CpuSet struct {
 }
 
 type Cpu struct {
-	Quota   string
-	Period  string
-	cpuStat map[string]string
+	Quota  string
+	Period string
+}
+
+type CpuStat struct {
```

Comment:
Each struct should represent a cgroup controller. i.e `cpu`,`cpuset`,`memory`,`pids`...
`stats` is not a controller, it's a field under the `cpu` controller
Could you please elaborate why this change is needed?

Reply:
We discussed this before stat has multi line values where the number files are different in v1 and v2 and with current implementation reading stat is difficult where the current implementation tries to read multiple files and thats a problem.

Reply:
@mrniranjan Can't you simply read the `cpu.stat` in a separate call and store it in `cfg.cpuStat`?
Add the logic here:
https://github.com/openshift/cluster-node-tuning-operator/blob/master/test/e2e/performanceprofile/functests/utils/cgroup/v1/v1.go#L45 
And here:
https://github.com/openshift/cluster-node-tuning-operator/blob/master/test/e2e/performanceprofile/functests/utils/cgroup/v2/v2.go#L45

something like:
```
cmd = []string{
		"/bin/cat",
		dirPath + "/cpu.stat",
	}
	b, err := pods.ExecCommandOnPod(cm.k8sClient, pod, containerName, cmd)
// do some parsing logic and store in map
```

Reply:
addressed in the latest commit